apps/platform/src/assets/config.json

```diff
@@ -7,7 +7,6 @@
   "appUrl": "https://biosimulations.org/",
   "apiUrl": "https://api.biosimulations.org/",
   "email": "info@biosimulations.org",
-  "docsUrl": "https://docs.biosimulations.org/",
```

Comment:
maybe we still need this (for use in hero-banner-util-button.component.html)??? ... or else, where does the hero button get the "docsUrl" config?

Reply:
resolved - moved this configuration to the appropriate 'hero' button config.